packages/sdk-v2/src/sdk/base/react.ts

```diff
@@ -191,39 +207,24 @@ export function useG$Balance(refresh: QueryParams["refresh"] = "never") {
         contract: gdxContract,
         method: "balanceOf",
         args: [account]
-      }
-    ].filter(_ => _.contract && chainId == SupportedChains.MAINNET),
-    { refresh: refreshOrNever, chainId: SupportedChains.MAINNET as unknown as ChainId }
+      },
+    ].filter(_ => _.contract && chainId == MAINNET),
+    { refresh: refreshOrNever, chainId: MAINNET as unknown as ChainId }
+  );
+
+  const [g$Value, goodValue, gdxValue] = [...results, mainnetGdx].map(
+    result => result && !result.error ? result.value[0] : undefined
   );
+  
+  const g$Balance = <CurrencyValue>useG$Amount(g$Value);
+  const goodBalance = <CurrencyValue>useG$Amount(goodValue, "GOOD");
+  const gdxBalance = <CurrencyValue>useG$Amount(gdxValue, "GDX");
```

Comment:
what is this syntax?
usually if you want to cast you use "as CurrencyValue"

Reply:
Just alternative cast syntax